src/error.js

```diff
@@ -534,6 +534,10 @@ export function getErrorReportData(
     data['spt'] = getMode().singlePassType;
   }
 
+  if (IS_ESM) {
+    runtime += '-esm';
```

Comment:
@jridgewell do you think this will overload the error group/tuple too much?

Reply:
I think this should just be `runtime = 'esm'`, that way it takes the place of 1p/3p, increasing the number of buckets by 50% instead of doubling it

Reply:
Either is fine.